hypothesis-python/src/hypothesis/internal/observability.py

```diff
@@ -123,6 +130,10 @@ def _system_metadata():
     }
 
 
+#: If ``False``, do not collect coverage information when observability is enabled.
+#:
+#: This is exposed for performance, as coverage collection can be slow on Python
+#: 3.11 and earlier. On Python 3.12+, there should be no need to set this.
```

Comment:
It's also useful to control data volume; for example I expect that we'll leave it disabled in hypofuzz!